openedx/core/djangoapps/credentials/management/commands/notify_credentials.py

```diff
@@ -217,7 +226,7 @@ def send_notifications(self, certs, grades, site_config=None, delay=0, page_size
             )
 
             user = User.objects.get(id=grade.user_id)
-            send_grade_if_interesting(user, grade.course_id, None, None, grade.letter_grade, grade.percent_grade)
+            send_grade_if_interesting(user, grade.course_id, mode, None, grade.letter_grade, grade.percent_grade)
```

Comment:
This set of changes isn't needed. If either mode or status is None, we get both from the DB. So we'll still look it up even with this change.